src/schema/v2/me/__tests__/myCollectionDeleteArtworkImageMutation.test.ts

```diff
@@ -0,0 +1,60 @@
+import { runAuthenticatedQuery } from "schema/v2/test/utils"
+import gql from "lib/gql"
+
+describe("MyCollectionDeleteArtworkImageMutation", () => {
+  const mutation = gql`
+    mutation {
+      MyCollectionDeleteArtworkImage(
+        input: { artworkId: "foo", imageId: "image1" }
+      ) {
+        artworkOrError {
+          ... on MyCollectionArtworkMutationDeleteSuccess {
+            success
+          }
+          ... on MyCollectionArtworkMutationFailure {
+            mutationError {
+              message
+            }
+          }
+        }
+      }
+    }
+  `
+
+  it("returns an error", async () => {
+    const context = {
+      deleteArtworkImageLoader: () =>
+        Promise.reject(
+          new Error(
+            `https://stagingapi.artsy.net/api/v1/my_collection/artworks/foo - {"error":"Error deleting artwork image"}`
+          )
+        ),
+    }
+
+    const data = await runAuthenticatedQuery(mutation, context)
+    expect(data).toEqual({
+      MyCollectionDeleteArtworkImage: {
+        artworkOrError: {
+          mutationError: {
+            message: "Error deleting artwork image",
+          },
+        },
+      },
+    })
+  })
+
+  it("deletes an artwork image", async () => {
```

Comment:
non-blocking: is this test name accurate? Is it demonstrating that an artwork image is actually deleted, or is it demonstrating the shape of the response?

Reply:
I just copied the delete artwork test.